ludwig/trainers/trainer_lightgbm.py

```diff
@@ -565,6 +566,7 @@ def train(
         save_path="model",
         **kwargs,
     ):
+        init_dist_strategy("local")
```

Comment:
Why is this not being executed by `LocalTrainingMixin.initialize()`? The distribution strategy initialization is intended to live in the backend, not the trainer.

Reply:
The pattern that I followed was how we call `init_dist_strategy` in `train_fn` [here](https://github.com/ludwig-ai/ludwig/blob/1529e02e0eff40182c7b607af530ab12aad42bca/ludwig/backend/ray.py#L172). We have an `initialize` function in `LocalTrainingMixin` and `HorovodBackend`. For the `RayBackend`, `initialize` just calls `ray.init`. I’ll move `init_dist_strategy` to be in backend.initialize functions.